commands/new.go

```diff
@@ -161,6 +161,9 @@ Use ` + "`hugo new [contentPath]`" + ` to create new content.`,
 The new site will have the correct structure, but no content or theme yet.
 Use ` + "`hugo new [contentPath]`" + ` to create new content.`,
 				run: func(ctx context.Context, cd *simplecobra.Commandeer, r *rootCommand, args []string) error {
+					if len(args) < 1 {
+						return errors.New("theme name needs to be provided")
```

Comment:
There's some logic above to distinguish between "command errors" (which we show the help output + an error message) and "runtime errors".

I think the above is in the first category, so if you could replace the above with

```go
return newUserError("theme name needs to be provided")
```

Reply:
Also, it would be good if you could adjust the commit message to be in line with the contribution guidelines.

Reply:
> There's some logic above to distinguish between "command errors" (which we show the help output + an error message) and "runtime errors".
> 
> I think the above is in the first category, so if you could replace the above with
> 
> ```go
> return newUserError("theme name needs to be provided")
> ```

Changed as requested. For the sake to consistency, I changed this at two other locations, too.